.vortex/installer/tests/Functional/FunctionalTestCase.php

```
@@ -70,9 +70,9 @@ protected function runNonInteractiveInstall(?string $dst = NULL, array $options
       $args['--' . $option] = $value;
     }
 
-    // Skip the database download in demo mode as it is not needed for the
+    // Skip the database fetch in demo mode as it is not needed for the
     // installer's tests.
-    Env::put(Config::IS_DEMO_DB_DOWNLOAD_SKIP, '1');
+    Env::put(Config::IS_DEMO_DB_FETCH_SKIP, '1');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Utils\Env' in method 'runNonInteractiveInstall'. (undefined)

(StaticAccess)